src/library_glfw.js

```diff
@@ -137,6 +137,8 @@ var LibraryGLFW = {
         case 0xAD:return 45; // DOM_VK_HYPHEN_MINUS -> GLFW_KEY_MINUS
         case 0xBE:return 46; // DOM_VK_PERIOD -> GLFW_KEY_PERIOD
         case 0xBF:return 47; // DOM_VK_SLASH -> GLFW_KEY_SLASH
+        case 0xBD:return 45; // Opera, IE, Safari, Chrome DOM_VK_MINUS -> GLFW_KEY_MINUS
```

Comment:
please keep the existing order, which is sorted by the return value. then this would be alongside `0xAD`, which makes sense as reading https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode I see that it parallels 0xBD, and we do indeed need both for full browser coverage.

According to the same link, equals needs 0xBB as you added, but also needs 0x3D, so let's add that. In fact we can remove the line with `0x61` - it looks like a typo, it goes to 61 but should have been from 0x3D all along.